src/ui/MLFacet/MLFacet.ts

```diff
@@ -113,10 +113,9 @@ export class MLFacet extends Component {
      *
      * **Minimum:** `0`
      *
-     * **Default:** `undefined`, and the default Search API value (`8`)
-     * is used.
+     * **Default:** `8`, which is the same as the Search API default value (`8`).
```

Comment:
If we enforce a default value in the UI, we might as well remove "`, which is the same as the Search API default value (`8`).